packages/manager/modules/pci/src/projects/project/notebooks/dashboard/dashboard.html

```diff
@@ -1,36 +1,61 @@
-<oui-header
-    data-heading="{{ $ctrl.notebook.spec.name }}"
-    description="{{ $ctrl.notebook.id }}"
->
-    <oui-header-tabs>
-        <oui-header-tabs-item
-            data-href="{{:: $ctrl.generalInformationLink }}"
-            data-active="$ctrl.currentActiveLink() === $ctrl.generalInformationLink"
-            ><span
-                data-translate="pci_notebooks_dashboard_info_tab_title"
-            ></span>
-        </oui-header-tabs-item>
-        <oui-header-tabs-item
-            data-href="{{:: $ctrl.attachDataLink }}"
-            data-active="$ctrl.currentActiveLink() === $ctrl.attachDataLink"
-            ><span
-                data-translate="pci_notebooks_dashboard_attach_data_tab_title"
-            ></span>
-        </oui-header-tabs-item>
-    </oui-header-tabs>
-    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
-        <oui-guide-menu-item
-            data-ng-repeat="guide in $ctrl.guideUrl.ai_notenooks track by $index"
-            data-href="{{:: guide.url }}"
-            data-on-click="$ctrl.trackClick($ctrl.guideTrackingSectionTags.ai_notebooks + guide.tracking)"
-            data-external
+<header class="oui-header">
+    <div class="oui-header__container">
+        <div class="oui-header__content">
```

Comment:
It's you don't have particular custom need, it's more efficient use the webComponent

Reply:
Again, sorry, forget to rebase, this code was for an old P, not linked to this one.